### PR TITLE
/userInvites.resend

### PR DESCRIPTION
**Pull request by dcsg**
## 🚨 wait until https://github.com/teamleadercrm/api/pull/268 is merged. Change base branch to master.

This will allow to resend the invite to the user again.

### CODE REVIEW

**Review by rathesDot — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

File: src/01-general/user-invites.apib [hunk 1]
```diff
@@ -32,3 +32,14 @@ Cancel a user invite.
...
+### userInvites.resend [GET /userInvites.resend]
+
+Resend a user invite.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `e88131bf-50c4-49d3-8ae3-47e5d9626bf6` (string, required)
+
++ Response 204
```
Comment (jenssegers) [inline src/01-general/user-invites.apib]: If this creates a new invitation this should be `201` with the new id in the response.

Comment (jenssegers) [inline src/01-general/user-invites.apib]: If this creates a new invitation this should be 201 with the new id in the response.

**Review by jenssegers — DISMISSED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -718,6 +721,22 @@ Cancel a user invite.
 
 + Response 204
 
+### userInvites.resend [GET /userInvites.resend]
+
+Resend a user invite.
```
Comment (jenssegers) [inline apiary.apib]: Should we add more information about what this does? That it cancels the previous invitation and creates a new one?

**Review by rathesDot — APPROVED** [review]

File: src/01-general/user-invites.apib [hunk 3]
```diff
@@ -32,3 +32,19 @@ Cancel a user invite.
         + id: `e88131bf-50c4-49d3-8ae3-47e5d9626bf6` (string, required)
 
 + Response 204
+
+### userInvites.resend [GET /userInvites.resend]
+
+Resend a user invite by cancelling the previous and creating a new invite.
```
Comment (jenssegers) [inline src/01-general/user-invites.apib]:
```suggestion
Resend a user invite by canceling the previous invite and creating a new one.
```

**Review by mrtus — APPROVED** [review]

File: src/01-general/user-invites.apib [hunk 4]
```diff
@@ -32,3 +32,19 @@ Cancel a user invite.
         + id: `e88131bf-50c4-49d3-8ae3-47e5d9626bf6` (string, required)
 
 + Response 204
+
+### userInvites.resend [GET /userInvites.resend]
+
+Resend a user invite by canceling the previous invite and creating a new one.
```
Comment (jenssegers) [inline src/01-general/user-invites.apib]: I thought we used US English but looks like we should use British English. So "cancelling" was correct!

```suggestion
Resend a user invite by cancelling the previous invite and creating a new one.
```

**Review by rathesDot — APPROVED** [review]

**Review by jenssegers — APPROVED** [review]

**Review by stivni — APPROVED** [review]

**Comment by JordyDeGroeve** [comment]
Anything blocking this from getting released?

**Pull request merged**